bot/exts/info/information.py

```diff
@@ -523,6 +525,40 @@ async def json(self, ctx: Context, message: Message) -> None:
         """Shows information about the raw API response in a copy-pasteable Python format."""
         await self.send_raw_content(ctx, message, json=True)
 
+    @command(aliases=("rule",))
+    async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
+        """Provides a link to all rules or, if specified, displays specific rule(s)."""
+        rules_embed = Embed(title="Rules", color=Colour.og_blurple(), url=f"{URLs.site_schema}{URLs.site}/pages/rules")
```

Comment:
Setting the URL here won't work in dev for people using Docker, since Discord's API does validation on embed URLs and deems `http://web:8000/pages/rules` an invalid url.

I wonder if it's *just fine* to hardcode this particular embed url to the prod site

Reply:
Don't see an issue with it if it may make using it locally a bit easier, some other places in the code hardcode the urls anyway, pushed in 1f2b603